samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/translation/MicrosoftTranslator.java

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.sample.multilingual.translation;
+
+import java.io.Reader;
+import java.io.StringReader;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionException;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.microsoft.bot.integration.Configuration;
+import com.microsoft.bot.sample.multilingual.translation.model.TranslatorResponse;
+
+import org.slf4j.LoggerFactory;
+
+import okhttp3.MediaType;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+
+/**
+ * A helper class wrapper for the Microsoft Translator API.
+ */
+public class MicrosoftTranslator {
+    private static final String HOST = "https://api.cognitive.microsofttranslator.com";
+    private static final String PATH = "/translate?api-version=3.0";
+    private static final String URI_PARAMS = "&to=";
+
+    private static OkHttpClient httpClient = new OkHttpClient();
+    private static String key;
+
+    /**
+     * @param configuration The configuration class with the translator key stored.
+     */
+    public MicrosoftTranslator(Configuration configuration) {
+        String translatorKey = configuration.getProperty("TranslatorKey");
+
+        if (translatorKey == null) {
+            throw new IllegalArgumentException("key");
+        }
+
+        MicrosoftTranslator.key = translatorKey;
+    }
+
+    /**
+     * Helper method to translate text to a specified language.
```

Comment:
Specified -> specific

Reply:
[I dont think it is a typo](https://www.quora.com/When-should-I-use-the-word-specific-vs-specified). Specified refers to a given language rather than specific.